python/setup.py

```diff
@@ -499,8 +499,9 @@ def get_platform_dependent_src_path(subdir):
          if int(version_major) >= 12 and int(version_minor1) >= 5 else subdir)(*version.split('.')))
 
 
+extension = sysconfig.get_config_var("EXE")
```

Comment:
nit: can we pick a more explicit name for this variable? Maybe `exe_extension`

Reply:
@ThomasRaoux changed